tools/lunar.repos

```diff
@@ -0,0 +1,49 @@
+# Use this underlay if building on Ubuntu 23.04.
+# https://github.com/ros/rosdistro/blob/master/rolling/distribution.yaml
+# Versions are usually manually synced with what rolling has, unless otherwise noted for some patches
+
+repositories:
```

Comment:
Is this not broadly identical to `underlay.repos`? Remove

Reply:
I looked at the history of this file, and it seemed it was temporary until upstream maintainers got their changes all the way into the debians. 

Since the forks I made have changes that are either in PR, or merged but not released, and not even in the debians, it's not clear how I can fold in these changes. Do you have any recommendations on what to do for these patches?

Reply:
One approach: Allow fallback to older target names: e3cda9c7d00f54110cdfadcb4a97f27cefa381ef